tools/encodings/arabic_unique-glyphs.nam

```diff
@@ -79,74 +79,29 @@
 0x0670 # ARABIC LETTER SUPERSCRIPT ALEF
 0x0671 # ARABIC LETTER ALEF WASLA
 0x0679 # ARABIC LETTER TTEH
-0x067A # ARABIC LETTER TTEHEH
```

Comment:
The removed characters seem to be for small and minority languages, so probably OK to drop from the default subset.